### PR TITLE
Upgrade to Tempo 2.2.1

### LINKED ISSUE #525: Support Tempo 2.2
Tempo 2.2 will go out this week. The RC is already out https://github.com/grafana/tempo/releases/tag/v2.2.0-rc.0

### PR DESCRIPTION
**Pull request by pavolloffay**
Resolves #525 

Release notes https://github.com/grafana/tempo/releases

Blocked by:

- [x] https://github.com/grafana/tempo/issues/2381 the tenant header configuration was removed

### CODE REVIEW

File: internal/manifests/config/tempo-config.yaml [hunk 1]
```diff
@@ -160,6 +160,8 @@ storage:
       path: /var/tempo/traces
     wal:
       path: /var/tempo/wal
+    block:
+      version: vParquet
```
[the remark this answers is not in the document]
Comment (pavolloffay) [reply internal/manifests/config/tempo-config.yaml]: It seems that 2.2. configured with parquetv2 format will be able to read parquetv1

**Comment by pavolloffay** [comment]
smoketest-with-jaeger  is failing. I will need to debug it locally

**Comment by pavolloffay** [comment]
The e2e tests pass when I disable internal mTLS. 

With enabled mTLS I am getting this error:

Screenshot of Jaeger UI (5)

cc) @rubenvp8510

**Comment by pavolloffay** [comment]
I have tried to set 
```
    tls_server_name: tempo-simplest-query-frontend.default.svc.cluster.local
    tls_cipher_suites: TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA38
```

to all components but no success to fix the TLS

**Review by andreasgerstmayr — APPROVED** [review]

**Pull request merged**